stactools_pipelines/pipelines/amazonia_1/config.yaml

```diff
@@ -1,9 +1,9 @@
 ---
   id: "amazonia_1"
   compute: "awslambda"
-  secret_arn: "arn:aws:secretsmanager:eu-central-1:751953039297:secret:asdi-auth-stack-alukach/asdi-workflows-6awSos"
+  secret_arn: "arn:aws:secretsmanager:us-west-2:751953039297:secret:asdi-auth-stack-dev/asdi-workflows-fSzAw7"
```

Comment:
Do we have an asdi-workflow secret in each region?

Reply:
Just us-west-2 and eu-central-1 from the looks of it. As long as the secret values are the same, there should be no impact as long as your lambda has permissions to retrieve the secret. These two secrets do differ though, one for the test stack